src/lib/api/group/client/groupApiClient.client.test.ts

```diff
@@ -1,7 +1,13 @@
 import { request } from '@/lib/api/common/client';
-import { createGroup } from '@/lib/api/group/client';
+import { postCreateGroup } from '@/lib/api/group/client';
 
-import { ICreateGroupApiResponse, ICreateGroupParams } from './groupApiClient.client';
+import { Err, Ok } from 'result-ts-type';
+
+import {
+  IPostCreateGroupApiResponse,
+  IPostCreateGroupPayload,
+  putRenameGroup,
+} from './groupApiClient.client';
```

Comment:
Combine the imports:
```ts
import { postCreateGroup } from '@/lib/api/group/client';
```
```ts
import {
  IPostCreateGroupApiResponse,
  IPostCreateGroupPayload,
  putRenameGroup,
} from './groupApiClient.client';
```

Reply:
You're right, and I also forgot to index the exports...
I fixed them!